### DISCUSSION TITLE
ngx-countdown timer inside joint js html template

### DISCUSSION THREAD

**Discussion by maja5627** [clientIO/joint#1785] [Q&A]
Hello Joint JS team I am wondering if there is a way I can add the ngx-countdown element to my jointjs html template. I have this template 

```
content: [
    '<div class="example">',
    '</div>'
].join('')
```
and I am trying to add this element 
`<countdown [config]="{ leftTime: 600, format: "mm:ss" }"></countdown> `
It is not a native html element but need to get it into my joint js html template like this 
```
content: [
    '<div class="example">',
          '<countdown [config]="{ leftTime: 600, format: "mm:ss" }"></countdown>',
    '</div>'
].join('')
```
Is this possible is joint js? I have been trying different things and trying to find some documentation but have not found anything that works. Please let me know if this is possible or if you have any recommendations thank you for the help!

**Comment by kumilingus** [comment]
You would have to take a similar approach to `Vue` and `React` examples, i.e. render and place the components over the JointJS paper.

- *Vue* - https://github.com/clientIO/joint/tree/master/demo/vuejs
- *React* - https://github.com/clientIO/joint-paper-react
- *JS* - https://resources.jointjs.com/demos/tasks
